### ISSUE TITLE
Mic sensitivity only start capturing voice level after joining a channel

### ISSUE THREAD

**Issue by sidneycrestani** [crmarsh/discord-linux-bugs#132]
I don't know if that's the default/desired behavior, but I think we should be able to setup our mic prior to joining.

**Comment by TheDoctorsLife** [comment]
I have also been witnessing this behavoir.

**Comment by crmarsh** [comment]
Yeah, this is how it works on all platforms, though I agree it is not really the desired behavior.
